src/metadata/ColumnMetadata.ts

```diff
@@ -355,15 +355,15 @@ export class ColumnMetadata {
             this.default = options.args.options.default;
         if (options.args.options.onUpdate)
             this.onUpdate = options.args.options.onUpdate;
-        if (options.args.options.scale !== null && options.args.options.scale !== undefined)
```

Comment:
are you sure about these checks? If I remember correctly @AlexMesser were added these two checks explicitly

Reply:
They were here because we allowed precision to be null previously(to work well with old mysql versions).
As for scale: it is defined as `scale?: number;` so only types allowed by typescript are number and undefined. We don't overwrite it anywhere(casting to any before writing), so we should be good.